### PR TITLE
Pulsating splash

### PR DESCRIPTION
**Pull request by evbambly**
Donor pulsating splash initialization.

Added pulse and adjusted mobile fit

image

### CODE REVIEW

File: packages/donor/src/screens/authentication/AuthLoadingScreen.tsx [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 import Logo from "../logo/Logo";
+import Styles from "./AuthLoadingScreen.module.scss";
 
 export default function AuthLoadingScreen() {
   return (
-    <div style={{ textAlign: "center" }}>
-      <Logo style={{ width: "100%", maxWidth: "320px", height: "400px" }} />
-      <Spinner />
+    <div className={Styles["logo"]}>
+      <Logo size={"large"} />
```
Comment (yaron1m) [inline packages/donor/src/screens/authentication/AuthLoadingScreen.tsx:7]: Why did you remove the spinner?

**Review by yaron1m — APPROVED** [review]

File: packages/donor/src/screens/authentication/AuthLoadingScreen.tsx (hunk 1 above)
Comment (evbambly) [reply packages/donor/src/screens/authentication/AuthLoadingScreen.tsx:7]: Unnecessary.
Looks more professional without.

Comment (yaron1m) [reply packages/donor/src/screens/authentication/AuthLoadingScreen.tsx:7]: @msmeravs Please confirm we want to remove the spinner from the splash screen
